redaxo/src/addons/structure/plugins/content/lib/article_content_base.php

```diff
@@ -294,6 +294,9 @@ public function getArticle($curctype = -1)
         if (0 != $this->getSlice) {
             $sliceLimit = ' AND ' . rex::getTablePrefix() . "article_slice.id = '" . ((int) $this->getSlice) . "' ";
         }
+        if ('edit' !== $this->mode) {
+            $sliceLimit .= ' AND ' . rex::getTablePrefix() . 'article_slice.status = 1';
```

Comment:
ggf. magic values vermeiden und konstanten anlegen?

Reply:
Finde ich in diesem Fall überflüssig, es ist ja aktuell eigentlich nur ein boolsches Feld, in der DB halt 0/1.